projects/npm-tools/packages/npm-scripts/src/scripts/createBridges.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * SPDX-FileCopyrightText: © 2019 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const {addNamespace} = require('@liferay/js-toolkit-core');
+const fs = require('fs');
+const path = require('path');
+
+module.exports = function (bridges, dir) {
+	const projectPackageJson = JSON.parse(
+		fs.readFileSync('package.json', 'utf-8')
```

Comment:
By convention, [`'utf8'`, not `'utf-8'`](https://nodejs.org/api/fs.html) (don't know if it still works, but we should do what the docs do).

Reply:
I suppose it still works for people like me :rofl:

Reply:
The thing is that node's convention is quite unfortunate, because the canonical name seems to be [UTF-8](https://en.wikipedia.org/wiki/UTF-8) :shrug: 

Anyway, the docs only refer to `utf8` as you said so I'm changing it.